src/image.rs

```diff
@@ -618,16 +618,15 @@ pub trait GenericImage: GenericImageView {
     /// In order to copy only a piece of the other image, use `sub_image`.
     ///
     /// # Returns
-    /// `true` if the copy was successful, `false` if the image could not
-    /// be copied due to size constraints.
+    /// Returns an error if the image is too large to be copied at the given position
     fn copy_from<O>(&mut self, other: &O, x: u32, y: u32) -> bool
```

Comment:
The `bool` here should actually be `ImageResult<()>`...

Reply:
Wow ! I messed this up when reopening the old PR.